src/libs/SidebarUtils.ts

```diff
@@ -49,7 +49,11 @@ Onyx.connect({
             (reportAction) => ReportActionsUtils.shouldReportActionBeVisibleAsLastAction(reportAction) && reportAction.actionName !== CONST.REPORT.ACTIONS.TYPE.CREATED,
         );
 
-        visibleReportActionItems[reportID] = reportActionsForDisplay[reportActionsForDisplay.length - 1];
+        const reportAction = reportActionsForDisplay.at(reportActionsForDisplay.length - 1);
```

Comment:
I guess it should do the same
```suggestion
        const reportAction = reportActionsForDisplay.at(-1);
```